src/beerocks/cli/beerocks_cli.cpp

```diff
@@ -65,8 +65,8 @@ void cli::resetArguments()
     command.clear();
     commandIt        = functionsMap.end();
     commandNumOfArgs = 0;
-    memset(args.intArgs, 0, ARGSNUM);
-    memset(args.floatArgs, 0, ARGSNUM);
+    memset(args.intArgs, 0, sizeof(args.intArgs));
+    memset(args.floatArgs, 0, sizeof(args.intArgs));
```

Comment:
shouldn't this be ```memset(args.floatArgs, 0, sizeof(args.floatArgs));```?

Reply:
Ouch, indeed... floats happen to be the same size as ints...